### ISSUE TITLE
Issues with FPC client sdk for go

### ISSUE THREAD

**Issue by mbrandenburger** [hyperledger/fabric-private-chaincode#623]
**Describe the bug**
It seems that the client SDK currently has two issues which prevent developer from integrating it in their application without cloning the FPC project.

* First, Latest FPC `release github.com/hyperledger/fabric-private-chaincode@v1.0.0-rc1` still contains a gitmodule, which path is `github.com/hyperledger-labs/fabric-private-chaincode` The following error appears:
```
github.com/hyperledger/fabric-private-chaincode@v1.0.0-rc1: parsing go.mod:
module declares its path as: github.com/hyperledger-labs/fabric-private-chaincode
	        but was required as: github.com/hyperledger/fabric-private-chaincode
```
To use the most recent version, the developer would need to run `go get github.com/hyperledger/fabric-private-chaincode@dadbd25` manually. However, once this is fixed the developer will run into the next issue .... 

* Second, the FPC go protos, which are needed by the FPC client, are not checked into the repo. This results in the following error:
```
github.com/hyperledger/fabric-private-chaincode/client_sdk/go/pkg/client/resmgmt imports github.com/hyperledger/fabric-private-chaincode/internal/protos: 
module github.com/hyperledger/fabric-private-chaincode@latest found (v1.0.0-rc1), but does not contain package github.com/hyperledger/fabric-private-chaincode/internal/protos
```

Instead of let go fetch the dependency automatically, the developer could use the local FPC repository where protos are generated using our make scripts in `$FPC_PATH/protos`.
Here an example for a modified `go.mod` file for an application that uses FPC 
```
module example.com/simple-go

go 1.16

replace github.com/hyperledger/fabric-private-chaincode => /Users/bur/Developer/gocode/src/github.com/hyperledger/fabric-private-chaincode

require (
	github.com/hyperledger/fabric v1.4.0-rc1.0.20201118191903-ec81f3e74fa1
	github.com/hyperledger/fabric-private-chaincode v0.0.0-20210826130630-dadbd252fb0f // indirect
	github.com/hyperledger/fabric-sdk-go v1.0.0
)

```

**To Reproduce**

- Go to `$FPC_PATH/samples/application/simple-go` and create a go module by `go mod init example.com/simple-go`.
- Next run `make` and see (the first) error as explained above.
- Fix that issue by `go get github.com/hyperledger/fabric-private-chaincode@dadbd25` to reference the latest commit on FPC.
- Run `make` again to see (the second) error as explained above.
- Modify `go.mod` to redirect to your local FPC repo where go protos are available.

**Expected behavior**

1. Latest release should have the correct go modules path (`github.com/hyperledger/fabric-private-chaincode`).
2. FPC repo should contain all required files to use the FPC client SDK as "normal" go dependency.

** Proposed solution **
The first issue will be resolved by pushing rc2 release.
The proto related issue can be solved by added the FPC `*.pb.go` into the repository.

**Comment by mbrandenburger** [comment]
The second issue was solved with #624 being merged.

**Comment by mbrandenburger** [comment]
With the latest release [v1.0-rc2](https://github.com/hyperledger/fabric-private-chaincode/releases/tag/v1.0.0-rc2) the problem discussed here is resolved.

When using the FPC SDK in a project, the latest FPC version can be fetched using `go get -u github.com/hyperledger/fabric-private-chaincode@latest`. 

See here a sample `go.mod`:
```
module example.com/simple-go

go 1.16

require (
	github.com/hyperledger/fabric v1.4.0-rc1.0.20201118191903-ec81f3e74fa1
	github.com/hyperledger/fabric-private-chaincode v1.0.0-rc2
	github.com/hyperledger/fabric-sdk-go v1.0.0
)
```
